snuba_sdk/query.py

```diff
@@ -1,24 +1,26 @@
 from dataclasses import dataclass, fields, replace
 from typing import Any, List, Optional, Sequence, Union
 
-from snuba_sdk.column import Column
-from snuba_sdk.conditions import BooleanCondition, Condition
-from snuba_sdk.entity import Entity
-from snuba_sdk.expressions import (
```

Comment:
Probably need to keep the original imports in this file

Reply:
Yeah I meant to do this in a test file, not here.